### ISSUE TITLE
Problem with node-config.yaml trying to run node.

### ISSUE THREAD

**Issue by Micro86** [input-output-hk/shelley-testnet#52]
**Describe the bug**
I am tying to run the node using: 
https://github.com/input-output-hk/shelley-testnet/wiki/How-to-setup-a-Jormungandr-Networking--node-(v-0.5.0)

Made a node-config.yaml file as shown in the How to setup.

After running command:
jormungandr --config node-config.yaml --genesis-block-hash adbdd5ede31637f6c9bad5c271eec0bc3d0cb9efb86a5b913bb55cba549d0770 --log-level=info

In powershell I keep getting: 
Error in the overall configuration of the node
Error while parsing the node configuration file: missing field `peer_2_peer` at line 2 column 4

I am sure I am making a mistake but can't figure out where.

**To Reproduce**
Run command

**Expected behavior**
Running node

**Additional context**
My node-config.yaml file:

---
log:
  format: plain
  level: info
  output: stderr
p2p:
  listen_address: /ip4/0.0.0.0/tcp/3100
  public_address: /ip4/192.168.192.55/tcp/3100
  topics_of_interest:
    blocks: high
    messages: high
  trusted_peers: [/ip4/3.123.177.192/tcp/3000,
                  /ip4/3.123.155.47/tcp/3000,
                  /ip4/52.57.157.167/tcp/3000,
                  /ip4/3.112.185.217/tcp/3000,
                  /ip4/18.140.134.230/tcp/3000,
                  /ip4/18.139.40.4/tcp/3000,
                  /ip4/3.115.57.216/tcp/3000]
rest:
  listen: "127.0.0.1:3101"
storage: /tmp/jormungandr

**Comment by rinor** [comment]
Probably config lines indentation. Copy paste the following AS IT IS and try again.

```
log:
  format: plain
  level: info
  output: stderr

p2p:
  listen_address: /ip4/0.0.0.0/tcp/3100
  public_address: /ip4/192.168.192.55/tcp/3100
  topics_of_interest:
    blocks: high
    messages: high
  trusted_peers: [
    /ip4/3.123.177.192/tcp/3000,
    /ip4/3.123.155.47/tcp/3000,
    /ip4/52.57.157.167/tcp/3000,
    /ip4/3.112.185.217/tcp/3000,
    /ip4/18.140.134.230/tcp/3000,
    /ip4/18.139.40.4/tcp/3000,
    /ip4/3.115.57.216/tcp/3000,
  ]
  
rest:
  listen: "127.0.0.1:3101"

storage: /tmp/jormungandr
```

**Comment by Micro86** [comment]
Hello thanks for your reply, sadly it didn't work. 
Now I am getting, it shifted to line 1:

Error in the overall configuration of the node
Error while parsing the node configuration file: missing field `peer_2_peer` at line 1 column 4

**Comment by rinor** [comment]
1) can you **attach** here the config file.
2) what is the output of `jormungandr --full-version`

**Comment by Micro86** [comment]
1. [node-config.txt](https://github.com/input-output-hk/shelley-testnet/files/3665617/node-config.txt) 
This is not the actual file because a .YAML file isn't supported for upload.

2. Output:

error: Found argument '--full-version' which wasn't expected, or isn't valid in this context
        Did you mean --version?

USAGE:
    jormungandr.exe --config <node_config> --version

For more information try --help

**Comment by Micro86** [comment]
[node-config.zip](https://github.com/input-output-hk/shelley-testnet/files/3665623/node-config.zip)

I made a .zip of node-config

**Comment by rinor** [comment]
You probably need to update your node.
check: `jormungandr --version`

**Comment by Micro86** [comment]
Yess thats probebly it, it is still version 0.3.0.
But I downloaded all the new files...

Is there an easy way to update it in cmd ??

**Comment by Micro86** [comment]
Issue solved, installed v0.5.2, node is working.

**Comment by dorin100** [comment]
Thank @rinor
